encrypted-media/scripts/playback-temporary-multikey-sequential.js

```diff
@@ -87,12 +87,13 @@ function runTest(config,qualifier) {
             // the new ready state is HAVE_CURRENT_DATA or less" case of the readyState change
             // algorithm which requires a "timeupdate" event be fired.
              if (_waitingForKey) {
-                assert_equals(++_timeupdateWhileWaiting, 1, "Should only receive one timeupdate while waiting for key");
-                assert_equals(_video.readyState, _video.HAVE_CURRENT_DATA, "Should not continue playing whilst waiting for a key");
+                assert_equals(++_timeupdateWhileWaitingCount, 1, "Should only receive one timeupdate while waiting for key");
+                assert_equals(_video.readyState, _video.HAVE_CURRENT_DATA, "Video readyState should be HAVE_CURRENT_DATA while wating for key");
             }
 
             if (_video.currentTime > config.duration) {
                 assert_equals(_mediaKeySessions.length, config.initData.length, "It should require all keys to reach end of content");
+                assert_equals(_timeupdateWhileWaitingCount, 1, "Should have only received one timeupdate while waiting for key");
```

Comment:
nit: ... received exactly one...

The check on line 90 checks no more than one. This checks that we did get that one.